extension/content.js

```diff
@@ -194,6 +215,8 @@ document.addEventListener('DOMContentLoaded', () => {
 	}
 
 	if (isRepo) {
+		const isRepoTree = window.location.href.split('/').length === 5 || window.location.href.split('/')[5] === 'tree';
```

Comment:
I don't think we should show it in `window.location.href.split('/')[5] === 'tree'` and I would call it `isRepoRoot` instead.